src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,55 +1,98 @@
-import React from 'react';
+import React, { FC, useState, ChangeEvent } from 'react';
 import './FindMovie.scss';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { hollowFilm } from '../../api/movies';
+import { getFilm } from '../../api/Api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+interface Props {
+  addMovie: (newMovie: Movie) => void;
+  movies: Movie[];
+}
+
+export const FindMovie: FC<Props> = (props) => {
+  const [movie, setMovie] = useState<Movie>(hollowFilm);
+  const [input, setInput] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setInput(event.target.value);
+    setErrorMessage('');
+  };
+
+  const findFilm = () => {
+    const response = getFilm(input);
+
+    response
+      .then(result => setMovie(result))
+      .catch(() => setErrorMessage('Can\'t find a movie with such a title'));
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const handleAddFilm = () => {
+    const isFilmInList = props.movies.find(item => item.imdbID === movie.imdbID) !== undefined;
+
+    if (movie !== hollowFilm && !isFilmInList) {
+      props.addMovie(movie);
+      setMovie(hollowFilm);
+      setInput('');
+    } else if (isFilmInList) {
+      setErrorMessage('This film is already in list');
+    }
+  };
+
+  return (
+    <>
+      <form className="find-movie">
```

Comment:
You should probably add onSubmit here and make preventDefault(), cause when i press 'Enter' with blank input, valid or invalid name of a movie it just reloads the page.